packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js

```diff
@@ -61,6 +65,11 @@ const ApplicationInfosPage = () => {
         if (menuLogo.rawFile) {
           trackUsage('didChangeLogo');
         }
```

Comment:
Did you check with Yannis, if they are aware that from now on, the number of tracking events might be higher, because two logos could have been updated? It may make sense to send as a payload which of them was changed ...